File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("elife-sciences_sdk")

describe("ElifeSciencesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { ElifeSciencesSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = ElifeSciencesSDK.test()
    equal(testsdk instanceof ElifeSciencesSDK, true,
      'ElifeSciencesSDK.test() must return a client synchronously')
  })

})
